.github/workflows/build.yml

```diff
@@ -0,0 +1,81 @@
+name: Build
+
+on: [push, pull_request]
+
+jobs:
+  compile:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Zeo++
+      run: |
+        git clone https://github.com/pzarabadip/zeoplusplus -b develop zeoplusplus
+        cd zeoplusplus
+        cd voro++/src
+        make
+        cd ../..
+        make
+        mkdir bin
+        cp network bin
+    - name: Upload network executable
+      uses: actions/upload-artifact@v1
+      with:
+        name: zeo
+        path: ./zeoplusplus/bin
+  tests:
+    needs: compile
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        python-version: [3.6,3.7,3.8]
+        backend: ['django','sqlalchemy']
+    steps:
+    - uses: actions/checkout@v1
+    - uses: harmon758/postgresql-action@v1
+      with:
+        postgresql version: '11'
+        postgresql db: test_${{ matrix.backend }}
+        postgresql user: 'postgres'
+        postgresql password: ''
```

Comment:
this can be deleted (we're using dynamically created psql cluster for tests)

Reply:
Done.